oaknational/Oak-Web-Application
# Feat/teacher search

## Description

- adds in tracking events for [teacher-search](https://www.avo.app/schemas/5PhajbVijwhXVKIJtGMT/branches/lt4Ht2V5v/diff) avo branch

## Issue(s)

Fixes #1474 

## How to test 
- There are 4 events in the link above for avo, check that the correct events are being to send to posthog

1. Go to {cloud link}
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
